apps/remix-ide/src/assets/js/loader.js

```diff
@@ -13,8 +13,8 @@ if (domains[window.location.hostname]) {
   _paq.push(['trackPageView']);
   _paq.push(['enableLinkTracking']);
   _paq.push(['enableHeartBeatTimer']);
-  if (!(window.localStorage.getItem('config-v0.8:.remix.config')) ||
-    ((window.localStorage.getItem('config-v0.8:.remix.config')) && !(window.localStorage.getItem('config-v0.8:.remix.config')).includes('settings/matomo-analytics'))) {
+  if (!window.localStorage.getItem('config-v0.8:.remix.config') ||
+    (window.localStorage.getItem('config-v0.8:.remix.config') && !window.localStorage.getItem('config-v0.8:.remix.config').includes('settings/matomo-analytics'))) {
```

Comment:
what is supposed to return  `window.localStorage.getItem('config-v0.8:.remix.config').includes('settings/matomo-analytics')` ? if it's a bool, you can't check it like this you have to use `undefined` or `null`